_pages/oidc/getting-started.md

```diff
@@ -59,16 +59,16 @@ The following implementation methods of OIDC are not supported by Login.gov for
 You are able to test authentication methods in real time with a testing account in our sandbox environment. To start, navigate to the [Login Partner Dashboard Sandbox](https://dashboard.int.identitysandbox.gov) and follow the steps below:
 
 - Select the “Sign-in” button to create a new account. Anyone with a .gov or .mil email address may request an account. 
-- Create a new team - see [Testing](https://developers.login.gov/testing/) page for instructions
+- Create a new team - see [Testing](https://developers.login.gov/testing/) page for instructions.
 -  Create a certificate - before creating your application you'll need to create a certificate that will be used to sign your requests. You can create a certificate using openssl. The example command to create the certificate from your terminal is:
     - `openssl req -nodes -x509 -days 365 -newkey rsa:2048 -keyout private.pem -out public.crt`
-- Create an application, at which point you will need to decide between private_key_jwt or PKCE
+- Create an application, at which point you will need to decide between private_key_jwt or PKCE.
 
 It is important to note that your Login.gov production account and your Login.gov sandbox account are two separate accounts.
 
 ### Auto-discovery
 
-Consistent with the specification, Login.gov provides a JSON endpoint for OpenID Connect auto-discovery at 
+Consistent with the specification, Login.gov provides a JSON endpoint for OIDC auto-discovery at 
```

Comment:
Personally I feel like this was clearer with the full name?
```suggestion
Consistent with the specification, Login.gov provides a JSON endpoint for OpenID Connect auto-discovery at 
```

Reply:
Formatting rules say always use the abbreviation after spelling it out but I think in this case clarity is more important than formatting so yes. I agree.